README.md

```diff
@@ -49,6 +49,29 @@ So have fun and start randomizing.
 In the [releases page](https://github.com/randovania/randovania/releases), we have zip files
 with everything ready to use. Just extract and run!
 
+## Linux
+
+### Running (in-source)
+
+1. Clone this repository (downloading the source zip is not supported and will not work)
+2. Open a terminal in the repository root
+3. Prepare the virtual environment and install dependencies:
+   
+   `$ tools/prepare_virtual_env.sh`
+5. Run the program from source:
+
+   `$ tools/start_client.sh`
+
+### Running Natively and Multiworld
+
+1. Activate the virtual environment and generate the necessary configuration.json, as well as build a native application:
+
+   `$ . venv/bin/activate`
+   `(venv) $ export PRODUCTION=true`
+   `(venv) $ python tools/create_release.py`
```

Comment:
I recommend a multiline code block with `console` highlighting:
```console
$ . venv/bin/activate
$ export PRODUCTION=true
$ python tools/create_release.py
```

Reply:
Updated with code blocks as well as added the instructions to checkout the latest release tag, as I forgot to do that.